docs/contributing.md

```diff
@@ -0,0 +1,64 @@
+- Please include a "risk assessment" in your PR descriptions describing what areas your PR could possibly affect. This helps reviewers less familiar with your code give a better review and anticipate problems.
+- Please CC the group Genomics-devs on your PRs, or the appropriate individual(s) from said group.  
+- If you're ready for a PR review make sure the PR is not marked as Draft.
+- Make sure all Azure changes work with a [Private VNet deployment](#setting-up-a-private-vnet). We're currently working on including this in integration testing so all verification must happen manually.
+
+## Developer Environment
+### Getting Started
+Instructions for getting your environment set up and building the executable can be found [here](https://github.com/microsoft/CromwellOnAzure#optional-build-the-executable-yourself). Once your build successfully deploys the Azure resources and runs a test workflow you're good to go. 
+
+If you run into any problems or want more information and ways to customize deployments and runs, please see the [Troubleshooting Guide](https://github.com/microsoft/CromwellOnAzure/blob/main/docs/troubleshooting-guide.md). 
+### So you're ready to contribute?
+You've read the above section on Issues and Pull Requests and you're ready to jump in! We ask that your contributions follow our coding standards and your changes work with a private vnet deployment.
+
+#### Coding Standards 
+Cromwell on Azure currently does not have enforced linting, so we do ask all contributors to help keep things neat. We generally follow the [C# Style Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions). 
+
+This includes things like 
+- Keeping one newline between end braces `}` 
+- Using `var` instead of explicitly typed variables
+- Keeping our `using` statements in alphabetical order (can be fixed by running Code Cleanup in Visual Studio)
+- Cleaning up unused variables
+- Making full use of modern C# conventions such as omitting type name or using object initalizer for the [`new` keyword](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/new-operator#constructor-invocation) and using modern operators such as [is](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/is) (ex: `vnetAndSubnet is not null` instead of `vnetAndSubnet != null`)
+- Using [Asynchronous Programming](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/async/) to effectively allocate resources and block as little as possible.
+
+We also ask contributors to please update the docs if you add a flag to Configuration.cs.
+
+#### Setting up a Private VNet
+(TODO: Replace this section with links to issue #286 once that goes in)
```

Comment:
Are you planning to leave this in for now?

Reply:
Yeah, it's note for when Jonathon finishes his document.